src/main/resources/assets/robotics/lang/no_no.json

```
@@ -0,0 +1,75 @@
+  "_": "->------------------------]  Spill Elementer  [------------------------<-",
```

Nothing big, there's a `{` missing here:
```suggestion
{
  "_": "->------------------------]  Spill Elementer  [------------------------<-",
```